controllers/shooter.py

```diff
@@ -46,12 +46,26 @@ def __init__(self) -> None:
         super().__init__()
         self.spin_command = False
         self.distance = None
+        self.disabled_flash: int = 0
 
     def execute(self) -> None:
         super().execute()
         self.update_LED()
 
     def update_LED(self) -> None:
+        # Flash if turret and shooter are disabled
+        if self.shooter.disabled:
```

Comment:
This comment seems to be wrong, the lights don't check if the turret is disabled

Reply:
They get disabled at the same time